### DISCUSSION TITLE
v2ray 下无法使用

### DISCUSSION THREAD

**Discussion by cauzp** [Richasy/Bili.Uwp#1210] [问答]
作者您好，我使用v2ray代理后，无法正常使用uwp程序，但是网页版b站是可以正常打开的，不知道是什么原因。

**Comment by tm0210** [comment]
https://github.com/Richasy/Bili.Uwp/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98

**Reply by tm0210** [reply]
点进去，里面有解决方案

**Comment by cauzp** [comment]
已成功解决，感谢
